spec/views/conferences/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "conferences/edit", type: :view do
+  before(:each) do
+    @conference = assign(:conference, Conference.create!(
+      :name => "MyString"
```

Comment:
Indent the first parameter one step more than `Conference.create!(`.<br>Use the new Ruby 1.9 hash syntax.